### PR TITLE
Magic Items Polished

### PR DESCRIPTION
**Pull request by Pvanduyse**

### CODE REVIEW

**Comment by jasonmcalpin** [comment]
You deleted the healing potions but I don't see them added anywhere else. What am I missing?

**Comment by maxrothman** [comment]
First of all, thanks again for all your hard work!

It seems that there are a lot of unnecessary changes in this PR, such as adding/deleting newlines at the beginning/end of files and changing the length of underlines. Were those changes intentional?

**Comment by Pvanduyse** [comment]
They may not make a difference on the website side of things, but when you look at the code, it now has exactly the same format for all items:
```
 .. _srd:item-name:

Item Name
------------------------------------------------------


*Type, Rarity*

Description.

```
All with the same type and amount of formatting, newlines, etc.. It means that flipping through the pages while looking at the code doesn't show things jumping around anymore.

As for the healing potions, If you check, you will find that "Healing Potions" (Notice the 's') is supposed to be a section under the "Healing Potion" item (Notice the lack of 's'). I moved the table that was under the "Healing Potions" page to the "Healing Potion" page, then deleted the duplicate.

**Comment by jasonmcalpin** [comment]
Okay great. I knew I was off somewhere. I didn't scroll down far enough. I see it now. Great job on these. Really appreciate the help!

File: source/equipment/magic-items/armor-of-resistance.rst [hunk 1]
```diff
@@ -10,5 +10,13 @@ Armor of Resistance
 You have resistance to one type of damage while you wear this armor. The
 GM chooses the type or determines it randomly from the options below.
 
-reaction to become the target of the attack instead.
```
Comment (maxrothman) [inline source/equipment/magic-items/armor-of-resistance.rst]: This line should be in "arrow-catching shield". Can you move it there?

File: source/equipment/magic-items/potion-of-healing.rst [hunk 2]
```diff
@@ -7,8 +7,21 @@ Potion of Healing
...
-points depends on the potion's rarity, as shown in the Potions of
-Healing table. Whatever its potency, the potion's red liquid glimmers
-when agitated.
+You regain hit points when you drink this potion. The
+number of hit points depends on the potion’s rarity, as
+shown in the Potions of Healing table. Whatever its
+potency, the potion’s red liquid glimmers when agitated.
+
+
+Potions of Healing
```
Comment (maxrothman) [inline source/equipment/magic-items/potion-of-healing.rst]: @eepMoody @jasonmcalpin should this be done as `..table:: Potions of Healing`? That's seems more semantic and that's how I did it in #126, but we don't seem to have styling for table titles yet. This is not a blocker for merging, we can go back and fix all the table titles later.

**Comment by maxrothman** [comment]
Ahhh, I see. Generally, this looks good. Once my comment(s) are addressed and the commits are squashed, I'll give my :+1:

File: source/equipment/magic-items/armor-of-resistance.rst (hunk 1 above)
Comment (Pvanduyse) [inline source/equipment/magic-items/armor-of-resistance.rst]: Done.

**Comment by Pvanduyse** [comment]
Since I'm not familiar to the table edits/formatting, can I recommend that these changes be merged, and that the tables be corrected in a different merge request?

**Comment by maxrothman** [comment]
FWIW: http://docutils.sourceforge.net/docs/ref/rst/directives.html#table

I think we should merge without it though. We'll go back and change them all once the styling is in place.

**Comment by eepMoody** [comment]
Just reviewed this after those fixes and it seems solid to me. :+1:

**Comment by maxrothman** [comment]
Closed in favor of #138

**Pull request closed without merging**